### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by tomereli**
Add issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+For example - `bml_conn_map` should show 2 radios, but it showing none.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Target (please complete the following information):**
+ - OS: [e.g. Linux DUMMY mode with ZMQ]
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion

 - Host: [e.g. Ubuntu 18.04]
 - Platform: [e.g. Linux DUMMY mode with ZMQ]
```

File: .github/ISSUE_TEMPLATE/feature-development.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+name: Feature development
+about: Feature currently being developed / reviewed
+title: "[FEATURE]"
+labels: enhancement
+assignees: ''
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/feature-development.md]: Shouldn't we add milestone here? And maybe default it to `Certifiable controller on Ubuntu NUC` (the milestone I added already).

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+title: "[BUG] <component>: <headline>"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Always mention the tag / hash on which this was tested AND how often it was reproduced (for example - 1/10 tries).
+
+**To Reproduce**
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Isn't it more appropriate to use a heading here?
```suggestion
## To Reproduce
```

Note that the **Describe the bug** doesn't need to be a heading IMO.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Steps to reproduce the behavior:
+1. Build
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: Your md-lint said that there should be an empty line before this :-)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+2. Run controller & Agent
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
2. Run controller & agent
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+3. See in the logs this and that / process stops responding / crashing / system behaves unexpectedly
+
+**Expected behavior**
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Expected behavior
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+A clear and concise description of what you expected to happen.
+For example - `bml_conn_map` should show 2 radios, but it showing none.
+
+**Screenshots**
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Screenshots
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+If applicable, add screenshots to help explain your problem.
+
+**Target (please complete the following information):**
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Target
**(please complete the following information)**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+ - OS: [e.g. Linux DUMMY mode with ZMQ]
+ - How you built prplMesh - full build command (for example `sudo ./build-image.sh && sudo ./build.sh -f MSGLIB=zmq CMAKE_BUILD_TYPE=Debug BUILD_TESTS=ON`
+ - How you ran prplMesh - for example `sudo build/install/scripts/prplmesh_utils.sh start`
+
+**Additional context**
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
## Additional context
```

File: .github/ISSUE_TEMPLATE/feature-development.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+---
+name: Feature development
+about: Feature currently being developed / reviewed
+title: "[FEATURE]"
```
Comment (arnout) [inline .github/ISSUE_TEMPLATE/feature-development.md]: I would call it a task. So
```
name: Task
title: "[TASK]"
```

**Review by arnout — CHANGES_REQUESTED** [review]
If we add this, we better get this right...

The commit subject is incorrect: it's not "Update", it's "Add".

CONTRIBUTING.md says that there should be a component, in this case I guess its "github".

**Comment by tomereli** [comment]
> If we add this, we better get this right...
> 
> The commit subject is incorrect: it's not "Update", it's "Add".
> 
> CONTRIBUTING.md says that there should be a component, in this case I guess its "github".

don't know how to fix the commit message without force push, so i'll do that at the final rebase before merge.

File: .github/ISSUE_TEMPLATE/feature-development.md (hunk 10 above)
Comment (tomereli) [reply .github/ISSUE_TEMPLATE/feature-development.md]: I changed this a lot, see fixup commit

**Review by arnout — APPROVED** [review]

**Pull request merged**